.github/workflows/preview-release.yml

```diff
@@ -42,6 +43,4 @@ jobs:
               run: |
                   git add .
                   git commit -am "chore: publish preview snapshot version"
-                  yarn prepublishOnly
                   yarn changeset publish --no-git-tag --tag preview
-                  git reset --hard HEAD^
```

Comment:
i thought we needed this for clean up

Reply:
Clean-up of local branches but this is a script running in CI so the entire environment will be thrown away at the end.